ext/gmp/gmp.c

```diff
@@ -521,6 +521,21 @@ static int gmp_unserialize(zval *object, zend_class_entry *ce, const unsigned ch
 }
 /* }}} */
 
+static void *gmp_alloc(size_t size)
+{
+	return emalloc(size);
+}
+
+static void *gmp_realloc(void *ptr, size_t old_size, size_t new_size)
+{
+	return erealloc(ptr, new_size);
+}
+
+static void gmp_free(void *ptr, size_t size)
+{
+	efree(ptr);
```

Comment:
pardon my likely irrelevant question. Not that familiar with mpz but is there never a compile time size allocation (e.g. mpq_init), just wondering if efree_size would be appropriate.

Reply:
Oh, I wasn't aware of `efree_size()`. Seems reasonable to use it. Thank you!